src/Commands/Parameters/StringLiteralParameter.ts

```diff
@@ -0,0 +1,23 @@
+import { Parameter } from "./Parameter";
+
+/**
+ * A string literal parameter.
+ */
+export class StringLiteralParameter extends Parameter {
+    /**
+     * The name of the string literal.
+     */
+    public name: string;
+
+    /**
+     * Initializes a new instance of the Parameter class.
+     * 
+     * @param descriptor   A plain-text description of the parameter.
```

Comment:
By docs I meant the documentation comments you're writing here. You put `descriptor` and `parameters` as the arguments in your comments, but the actual argument names are `name` and `description`.